src/main/kotlin/org/codefreak/codefreak/service/TaskService.kt

```diff
@@ -38,73 +26,53 @@ TaskService : BaseService() {
   private lateinit var assignmentRepository: AssignmentRepository
 
   @Autowired
-  private lateinit var evaluationService: EvaluationService
-
-  @Autowired
-  private lateinit var evaluationStepDefinitionRepository: EvaluationStepDefinitionRepository
-
-  @Autowired
-  private lateinit var fileService: FileService
-
-  @Autowired
-  @Qualifier("yamlObjectMapper")
-  private lateinit var yamlMapper: ObjectMapper
+  private lateinit var taskTarHelper: TaskTarHelper
```

Comment:
Maybe you should call all methods directly on the `TaskTarHelper` instead of proxying them, so you can drop this dependency and prevent a circular dependency between `TaskService` and `TaskTarHelper`.